ci-operator/step-registry/rosa/tests/e2e/fullcycle/rosa-tests-e2e-fullcycle-chain.yaml

```diff
@@ -4,6 +4,8 @@ chain:
   - ref: rosa-tests-runtime-day1-post
   - ref: rosa-tests-runtime-day2
   - ref: rosa-tests-runtime-destructive
+  - ref: rosa-tests-runtime-destroy
+  - ref: rosa-tests-runtime-destroy-post
```

Comment:
Will this destroy-post be a standard test in the whole test process?

Reply:
@yasun1 I think so. the destroy-post testing will be used for all jobs which created cluster by profile

Reply:
Then it should be called after the `rosa-teardown` post step from `rosa-lifecycle` worflow ?
Or create a new workflow ?

Reply:
I fear this would create problems in other jobs ?

Reply:
@radtriste destroy and destroy-post step in rosa-lifecycle , rather then after rosa-teardown is for reporting the result. I have tested on local to run rosa-tests-runtime-destroy twice. The rosa-teardown step can be as a double confirm to delete the resource. 
Could you explain more about your concern that "this would create problems in other jobs"?